_api-reference/list/list-indices.md

```diff
@@ -35,30 +35,47 @@ Parameter | Type | Description
 `time` | Time | Specifies the time units, for example, `5d` or `7h`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
 `expand_wildcards` | Enum | Expands wildcard expressions to concrete indexes. Combine multiple values with commas. Supported values are `all`, `open`, `closed`, `hidden`, and `none`. Default is `open`.
 `next_token` | String | Fetches the next page of indexes. When `null`, only provides the first page of indexes. Default is `null`. 
-`size` | Integer | The maximum number of indexes to be displayed on a single page. The number of indexes on a single page of the response is not always equal to the specified `size`. Default is `500`. Maximum value is `5000`.
+`size` | Integer | The maximum number of indexes to be displayed on a single page. The number of indexes on a single page of the response is not always equal to the specified `size`. Default is `500`. Minimum is `1` and maximum value is `5000`.
 `sort` | String | The order in which the indexes are displayed. If `desc`, then the most recently created indexes are displayed first. If `asc`, then the oldest indexes are displayed first. Default is `asc`.
 
 ## Example requests
 
+To get information for all the indexes, use the following query and keep specifying the `next_token` as received from response until its `null`:
+
+```json
+GET _list/indices/<index>?v&next_token=token
+```
+
+
 To limit the information to a specific index, add the index name after your query, as shown in the following example:
 
 ```json
 GET _list/indices/<index>?v
 ```
 {% include copy-curl.html %}
 
-If you want to get information for more than one index, separate the indexes with commas, as shown in the following example:
+If you want to get information for more than one index, separate the indexes with commas, as shown in the following example and keep using the `next_token` as received from response until its `null`:
```

Comment:
@gargharsh3134: What do you mean by "keep using the `next_token` as received by the response until its `null`? In my testing `next_token` wasn't required.

Reply:
Lets add it as a note under the example. I'll go ahead and update it and ping you once I'm done.